vulnerabilities/caching-storage-in-memory.md

```diff
@@ -0,0 +1,126 @@
+## Caching Storage in Memory
```

Comment:
Wdyt about instead naming it, "Unsafe storage caching"? IMO the current name seems to imply that it's generally unsafe to cache storage in memory

Note that if we change the name, we should also change the filename and the reference in the readme